perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +887,25 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+  benchmark_spec.mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
+  benchmark_spec.mysql_svc_oltp_table_size = FLAGS.mysql_svc_oltp_table_size
+
   # Prepare service specific states (create DB instance, configure it, etc)
   MYSQL_SERVICE_BENCHMARK_DICTIONARY[FLAGS.cloud].Prepare(vms[0])
 
+  metadata = {
+      'mysql_svc_oltp_tables_count': benchmark_spec.mysql_svc_oltp_tables_count,
+      'mysql_svc_oltp_table_size': benchmark_spec.mysql_svc_oltp_table_size,
+      'mysql_svc_db_instance_cores': FLAGS.mysql_svc_db_instance_cores,
+      'sysbench_warm_up_seconds': FLAGS.sysbench_warmup_seconds,
+      'sysbench_run_seconds': FLAGS.sysbench_run_seconds,
+      'sysbench_thread_count': FLAGS.sysbench_thread_count,
+      'sysbench_latency_percentile': FLAGS.sysbench_latency_percentile,
+      'sysbench_report_interval': FLAGS.sysbench_report_interval
+  }
+  DATA_LOAD_RESULTS.append(_PrepareSysbench(vms[0], metadata))
```

Comment:
[note] As found in test, this should be assignment ('='). Otherwise DATA_LOAD_RESULTS becomes a list of list.